src/main/java/ro/vladutit/Don/t/forget/v2/service/CategoryService.java

```diff
@@ -36,4 +36,13 @@ public Category getCategoryById(Long id) {
     public void deleteCategoryById(Long id) {
         categoryRepository.deleteById(id);
     }
+
+    public int getCategoryByName(String name) {
+        Optional<Category> optional = categoryRepository.findByNameIgnoreCase(name);
+        if(optional.isPresent()) {
+            return 1;
+        } else {
+            return 0;
```

Comment:
why do you need if..else if your function returns something ?????

Reply:
Very good point.